.github/workflows/benches.yml

```
@@ -54,22 +54,22 @@ jobs:
       - run: echo "$HOME/.cargo/bin" >> $GITHUB_PATH
       - run: bash scripts/environment/prepare.sh
       - run: echo "::add-matcher::.github/matchers/rust.json"
-      - name: Download baseline benchmarks
-        uses: dawidd6/action-download-artifact@891cccee4b25d3306cf5edafa174ddc1d969871f
-        continue-on-error: true
-        with:
-          github_token: ${{secrets.GITHUB_TOKEN}}
-          workflow: benches.yml
-          branch: master
-          name: criterion
-          path: target
-          workflow_conclusion: success
-      - name: Check if there were master artifacts
-        run: |
-          test -f target/criterion.zip || echo "::warning::No master benchmark artifacts could be fetched for comparison."
-      - run: unzip target/criterion.zip
-        continue-on-error: true
       - run: make slim-builds
+
+      - name: Checkout master to build first
+        run: |
+          git fetch --depth 1 origin master
+          git checkout origin/master
+        # build benchmarks on all CPUs, including isolated benchmarking CPU
+      - name: Prebuild master benchmarks
+        run: taskset -c "0-$(nproc)" make bench CARGO_BENCH_FLAGS="--no-run"
```

```suggestion
        run: taskset -c "0-$(nproc)" make bench-all CARGO_BENCH_FLAGS="--no-run"
```

Why isn't it using `bench-all`? Also, why `taskset` here?

---

Oops, I forgot to update to use `bench-all` after the merge from master. Thanks for catching!

I'm using `taskset` here since CPU 0 is isolated on the benchmark runners, but we'd like to use it during the build of the benchmarks.